riff-raff/app/persistence/ContinuousDeploymentConfigRepository.scala

```diff
@@ -3,18 +3,18 @@ package persistence
 import java.util.UUID
 import ci.{ContinuousDeploymentConfig, Trigger}
 import org.scanamo.syntax._
-import org.scanamo.auto._
+import org.scanamo.generic.auto._
 import org.scanamo.{DynamoFormat, Table}
-import cats.syntax.either._
 import conf.Config
 
 class ContinuousDeploymentConfigRepository(config: Config)
     extends DynamoRepository(config) {
 
   implicit val triggerModeFormat =
     DynamoFormat.coercedXmap[Trigger.Mode, String, NoSuchElementException](
-      Trigger.withName
-    )(_.toString)
+      Trigger.withName,
+      _.toString
+    )
```

Comment:
`DynamoFormat.coercedXmap()` now has a single set of two arguments, rather than curried arguments (due to https://github.com/scanamo/scanamo/pull/588 - released with [1.0-M13](https://github.com/scanamo/scanamo/releases/tag/v1.0-M13))